ivadomed/config/config_spineGeHemis.json

```diff
@@ -73,8 +73,8 @@
         "ram": true,
         "hdf5_path": "/home/andreanne/Documents/dataset/HeMIS.hdf5",
         "csv_path": "/home/andreanne/Documents/dataset/HeMIS.csv",
-        "target_lst": ["T2w"],
-        "roi_lst": null
+        "target_contrast": "T2w",
```

Comment:
Now, only one contrast can be selected as GT for `target_contrast` and `roi_contrast`.